ocaml/lambda/translprim.ml

```diff
@@ -180,6 +182,108 @@ let to_lambda_prim prim ~poly_sort =
+        fun ~unsafe ~boxed:_ ~mode -> Pbytes_load_f32 (unsafe, mode) );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_bytes_get64%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode -> Pbytes_load_64 (unsafe, mode) );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_bytes_getu128%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode -> Pbytes_load_128 { unsafe; mode } );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_bytes_set16%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pbytes_set_16 unsafe );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_bytes_set32%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pbytes_set_32 unsafe );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_bytes_setf32%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pbytes_set_f32 unsafe );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_bytes_set64%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pbytes_set_64 unsafe );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_bytes_setu128%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pbytes_set_128 { unsafe } );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_string_get16%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pstring_load_16 unsafe );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_string_get32%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode -> Pstring_load_32 (unsafe, mode) );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_string_getf32%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode -> Pstring_load_f32 (unsafe, mode) );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_string_get64%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode -> Pstring_load_64 (unsafe, mode) );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_string_getu128%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode -> Pstring_load_128 { unsafe; mode } );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_string_set16%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pbytes_set_16 unsafe );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_string_set32%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pbytes_set_32 unsafe );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_string_setf32%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pbytes_set_f32 unsafe );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_string_set64%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pbytes_set_64 unsafe );
+      ( (fun unsafe _boxed -> Printf.sprintf "%%caml_string_setu128%s" unsafe),
+        fun ~unsafe ~boxed:_ ~mode:_ -> Pbytes_set_128 { unsafe } );
+    ]
+  in
+  (let ( let* ) x f = List.concat_map f x in
+   let* string_gen, primitive_gen = types_and_widths in
+   let* unsafe, unsafe_sigil = [ (true, "u"); (false, "") ] in
+   let* boxed, boxed_sigil = [ (true, ""); (false, "#") ] in
+   let string = string_gen unsafe_sigil boxed_sigil in
+   let primitive = primitive_gen ~unsafe ~boxed in
+   let arity =
+     let is_substring ~substring string =
```

Comment:
`is_substring` seems like it should be factored out

Reply:
Moved to `Misc.Stdlib.String`.